examples/chart/teleport-daemonset/templates/daemonset.yaml

```diff
@@ -0,0 +1,142 @@
+        - mountPath: /host/usr/local/bin
+          name: {{ template "teleport.fullname" . }}-daemonset-host-usr-local-bin
+        - mountPath: /var/log/teleport.log
+          name: {{ template "teleport.fullname" . }}-daemonset-host-var-log-teleportlog
+        - mountPath: /var/run/dbus
+          name: {{ template "teleport.fullname" . }}-daemonset-host-var-run-dbus
+{{- if .Values.daemonset.extraVolumeMounts }}
+{{ toYaml .Values.daemonset.extraVolumeMounts | indent 8 }}
+{{- end }}
+      volumes:
+{{- if .Values.license.enabled }}
+      - name: {{ template "teleport.fullname" . }}-license
+        secret:
+          secretName: {{ .Values.license.secretName }}
+{{- end }}
+      - name: {{ template "teleport.fullname" . }}-bootstrap-scripts
+        configMap:
+          name: {{ template "teleport.fullname" . }}-bootstrap-scripts
+      - name: {{ template "teleport.fullname" . }}-daemonset-config
+        configMap:
+          name: {{ template "teleport.fullname" . }}-daemonset-config
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-config
+        hostPath:
+          path: /etc/teleport
+          type: DirectoryOrCreate
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-etc-systemd-system
+        hostPath:
+          path: /etc/systemd/system
+          type: Directory
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-run
+        hostPath:
+          path: /run
+          type: Directory
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-storage
+        hostPath:
+          path: /var/lib/teleport
+          type: DirectoryOrCreate
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-sys-fs-cgroup
+        hostPath:
+          path: /sys/fs/cgroup
+          type: Directory
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-sys-fs-cgroup-systemd
+        hostPath:
+          path: /sys/fs/cgroup/systemd
+          type: Directory
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-usr-local-bin
+        hostPath:
+          path: /usr/local/bin
+          type: DirectoryOrCreate
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-var-log-teleportlog
+        hostPath:
+          path: /var/log/teleport.log
+          type: FileOrCreate
+      - name: {{ template "teleport.fullname" . }}-daemonset-host-var-run-dbus
+        hostPath:
+          path: /var/run/dbus
+{{- if .Values.extraVolumes }}
+{{ toYaml .Values.extraVolumes | indent 6 }}
+{{- end }}
+{{- if .Values.image.pullSecrets }}
+      imagePullSecrets:
+{{ toYaml .Values.image.pullSecrets | indent 6 }}
+{{- end }}
+{{- if .Values.affinity }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}
+{{- end }}
+{{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 6 }}
+{{- end }}
+      serviceAccountName: {{ template "teleport.serviceAccountName" . }}
```

Comment:
also add namespace:
`namespace: {{ .Release.Namespace }}`

Reply:
The jury seems to still be out on whether Helm charts should actually specify a namespace or not - https://github.com/helm/helm/issues/5465

I think the recommendation is if you want to install a Helm chart into a particular namespace, that needs to be specified with the `--namespace` flag to `helm install`. `{{ Release.Namespace }}` is more typically used to make sure that you correctly describe namespaces when using RBAC via  `ClusterRoleBinding` etc.